PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py

```diff
@@ -42,37 +42,35 @@ def applyDeepBtagging( process, postfix="" ) :
     # delete module not used anymore (slimmedJets substitutes)
     delattr(process, 'selectedUpdatedPatJetsSlimmedDeepFlavour'+postfix)
 
-
     #switch to True if needed for private tests
-    enableDoubleB = False
-    if enableDoubleB:
-        # update slimmed jets to include DeepFlavour (keep same name)
-        # make clone for DeepDoubleB-less slimmed AK8 jets, so output name is preserved
-        addToProcessAndTask('slimmedJetsAK8NoDeepDoubleB', process.slimmedJetsAK8.clone(), process, task)
+    enableDeepTags = False
+    if enableDeepTags:
+        from RecoBTag.MXNet.pfDeepBoostedJet_cff import _pfDeepBoostedJetTagsAll as pfDeepBoostedJetTagsAll
+        # update slimmed jets to include particle-based deep taggers (keep same name)
+        # make clone for DeepTags-less slimmed AK8 jets, so output name is preserved
+        addToProcessAndTask('slimmedJetsAK8NoDeepTags', process.slimmedJetsAK8.clone(), process, task)
         updateJetCollection(
-            process,
-            jetSource = cms.InputTag('slimmedJetsAK8NoDeepDoubleB'),
-            # updateJetCollection defaults to MiniAOD inputs but
-            # here it is made explicit (as in training or MINIAOD redoing)
-            pvSource = cms.InputTag('offlineSlimmedPrimaryVertices'),
-            pfCandidates = cms.InputTag('packedPFCandidates'),
-            svSource = cms.InputTag('slimmedSecondaryVertices'),
-            muSource = cms.InputTag('slimmedMuons'),
-            elSource = cms.InputTag('slimmedElectrons'),
-            rParam = 0.8,
-            jetCorrections = ('AK8PFPuppi', cms.vstring(['L1FastJet', 'L2Relative', 'L3Absolute']), 'None'),
-            btagDiscriminators = [
-                'pfDeepDoubleBJetTags:probQ',
-                'pfDeepDoubleBJetTags:probH',
-                ],
-            postfix = 'SlimmedAK8DeepDoubleB'+postfix,
-            printWarning = False
-            )
-        
-        # slimmedJetsAK8 with DeepDoubleB (remove DeepDoubleB-less)
+           process,
+           jetSource=cms.InputTag('slimmedJetsAK8NoDeepTags'),
```

Comment:
here and below please keep spaces as in the other CMSSW versions (```jetSource = cms.InputTag('slimmedJetsAK8NoDeepTags'),```)

Reply:
@slava77 Fixed in https://github.com/cms-sw/cmssw/pull/24506/commits/cdd89f81800d6a4e6f7f629293ff009963dea7ca.